### PR TITLE
Publish scripts

### PR DESCRIPTION
**Pull request by pimeys**
# Build

```bash
nix build
```

# Update version

```bash
nix run .#updatePackageVersion 0.1.14
```

# Publish to crates.io

```bash
nix run .#publishRust
```

# Publish to npmjs.com

```bash
nix run .#publishJavascript
```

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    in {
+      defaultPackage = rustPlatform.buildRustPackage {
+        name = "connection-string";
+        src = builtins.path { path = ./.; name = "connection-string"; };
+
+        cargoLock = {
+          lockFile = ./Cargo.lock;
+        };
+
+        nativeBuildInputs = [ rust wasm-bindgen-cli nodejs ];
```
Comment (tomhoule) [inline flake.nix]: nitpick: do we need nodejs as a build input here? I don't see where

**Review by tomhoule — APPROVED** [review]
smooth!

File: flake.nix (hunk 1 above)
Comment (pimeys) [reply flake.nix]: nope.

Comment (pimeys) [reply flake.nix]: removed it and added a devshell

**Pull request merged**